doc/qbk/02_10_init_lists.qbk

```diff
@@ -11,7 +11,7 @@
 
 [section:initializer_lists Initializer Lists]
 
-A __value__ can be represented using code with C++ initializer lists:
+A __value__ can be represented in code with C++ initializer lists:
```

Comment:
Initializer lists may be used to construct or assign objects of type __value__, __object__, and __array__. Here we construct a `value`: